mountaineer/__tests__/actions/test_sideeffect_dec.py

```diff
@@ -83,7 +83,7 @@ async def mock_get_render_parameters(*args, **kwargs):
     # After our wrapper is called, our function is now async
     # Avoid the dependency resolution logic since that's tested separately
     with patch(
-        "mountaineer.actions.sideeffect.get_render_parameters"
```

Comment:
Module resolution order changed in 3.11. Back in 3.10 this patch failed because the interpreter confused `sideeffect` the module with `sideeffect` the function that is exported at the main `mountaineer.actions` level. We change the name of this module to avoid this conflict. Clients should only be importing actions from the root `mountaineer` package or maybe `mountaineer.actions`. These files shouldn't be indexed into specifically.